apps/template-studio/src/styles.css

```
@@ -1,35 +1,496 @@
+	--ts-text-muted: #aba6a0;
+	--ts-text-label: #8a7e76;
+	--ts-accent: #c9844a;
+	--ts-accent-soft: #f1deca;
+	--ts-accent-strong: #a26733;
+	--ts-danger: #b94a3b;
+	--ts-success: #5d7d4f;
+	--ts-radius: 10px;
+	--ts-radius-sm: 6px;
+	--ts-shadow-paper: 0 2px 4px rgba(40, 32, 22, 0.08), 0 18px 40px rgba(40, 32, 22, 0.12);
+	--ts-shadow-panel: 0 1px 2px rgba(40, 32, 22, 0.04);
+	font-family: Inter, ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', sans-serif;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)